Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -325,9 +349,12 @@ bool WebXROpaqueFramebuffer::setupFramebuffer()
                 gl.framebufferRenderbuffer(GL::FRAMEBUFFER, GL::DEPTH_ATTACHMENT, GL::RENDERBUFFER, m_depthStencilBuffer);
             }
         }
+
+        return gl.checkFramebufferStatus(GL::FRAMEBUFFER) == GL::FRAMEBUFFER_COMPLETE;
     }
 
-    return true;
+    // FIXME: It seems unlikely we'd ever get to this point.
+    return false;
```

Comment:
Yeah. I actually was tempted to put an ASSERT in there just to see if there ever was a situation we'd hit this.

Reply:
It seems like an `ASSERT_NOT_REACHED()` is in order.

Reply:
I'm not sure if other ports ever get into this state. I guess we'll find out because I just `return false;`ed them :). I'll add the `checkFramebufferStatus` though.